core/src/main/java/io/questdb/cairo/mv/MatViewRefreshExecutionContext.java

```diff
@@ -104,6 +104,11 @@ public TableReader getReader(TableToken tableToken) {
         return getCairoEngine().getReader(tableToken);
     }
 
+    @Override
+    public boolean isDeterministic() {
```

Comment:
nit: I'd make it the same naming as functions with non-deterministic wording, e.g. `allowNonDeterministic`. The default value of `true` is also more obvious for `allow` property.

Reply:
Renamed in 245eb3b35c7cc5589c94c1384105ea28d71b7061